apps/extension/src/core/handlers/Extension.ts

```diff
@@ -97,12 +101,43 @@ export default class Extension extends ExtensionHandler {
       DEBUG || TEST ? 0 : 2000
     )
 
+    // setup polling for config from github
+    let CONFIG_UPDATE_INTERVAL = 1000 * 60 * 5 // 5 minutes
+    const MAX_CONFIG_UPDATE_INTERVAL = 1000 * 60 * 60 // 1 hour
+    setInterval(() => {
+      try {
+        this.getRemoteConfig()
```

Comment:
async function here, errors woudln't be caught.
might want to use prom.catch() pattern